README.md

```diff
@@ -30,9 +27,9 @@ Developing for the cloud today requires mastering various layers of cloud stack,
 
 We fulfill our mission by reducing cognitive load and increasing iteration speeds. How? by enabling you to work at a higher level of abstraction, letting you focus on business logic instead of cloud mechanics, and write much less code. We also provide you with a set of tools that let you test your code locally, significantly faster than before.
 
-<a href="https://youtu.be/vHy1TM2JzUQ" target="_blank" align="left">
-    <img src="./logo/demo.gif" alt="Wing Demo" height="300px">
-</a>
+<p align="center">
+  <img src="./logo/demo_v2.gif" alt="Wing Demo" height="360px" width="640px">
```

Comment:
```suggestion
  <img src="./logo/demo.gif" alt="Wing Demo" height="360px">
```

1. Since you're deleting the original gif, might as well drop the v2
2. the img should maintain aspect ration afaik, so should be fine to pick either height or width to specify